package.json

```diff
@@ -146,5 +145,8 @@
       "default": "./dist/rollup.js"
     },
     "./dist/": "./dist/"
+  },
+  "dependencies": {
+    "fork-acorn-optional-chaining": "^7.2.0"
```

Comment:
Please no unnecessary external dependencies. Also why do we need a fork, is it impossible to do this via plugin? I do not feel good about disconnecting from core acorn updates for an unspecified time.

Reply:
The implementation at https://github.com/acornjs/acorn/pull/891/files is provided as a PR not a plugin. It might be possible to convert the implementation into a plugin, I don't have the resources for that currently though unfortunately.